backend/dynamic-ui/http-data-modelling.ftd

```diff
@@ -0,0 +1,255 @@
+-- import: fifthtry.github.io/admonitions as cbox
+
+-- ds.page: Basics of `http` processor and `data modelling`
+
+In this part, we will delve into essential concepts like the `http` processor
+and `data modelling` through a concise yet illuminating example.
+
+To facilitate your learning journey, we will take a pratical scenario involving 
+an array of records. Each record has two key attributes: the **country-name** 
+and it's corresponding **capital**, in the form of JSON data.
+
+-- ftd.text: Click to see the JSON data
+link: https://famous-loincloth-ox.cyclic.app/
+open-in-new-tab: true
+
+-- ds.markdown:
+
+By the end of this part, you will have gained insights into how REST APIs can 
+seemlessly connect the backend with the frontend.
+
+The first step is to create a `fastn package`.
+
+-- cbox.info: What is `fastn` package?
+
+`fastn` package is a folder that requires atleast two files
+
+- FASTN.ftd
+- index.ftd
+
+-- ds.h2: http processor
+
+`http` processor does the network communication using REST API and fetches data
+from the external site in a different domain.
+
+The syntax of using `http` processor is as follows:
+
+-- ds.code:
+
+\$processor$: pr.http
+
```

Comment:
This is wrong. Use proper snippet, 

```
-- record r:
$processor$: foo
```

`$processor$` on its own is nonsense.

Reply:
Also there is an import statement you have omitted.

Reply:
Updated 

```
\-- import: fastn/processors as pr  ;; <hl>

\-- record r:
\$processor$: http.pr  ;; <hl>
```